source/features/repo-age.tsx

```diff
@@ -30,21 +30,23 @@ const getFirstCommitDate = cache.function(async (): Promise<string | undefined>
 	}
 
 	if (commitsCount === 1) {
-		return select('.commit-tease-sha + span relative-time')!.attributes.datetime.value;
+		return [select('.commit-tease-sha + span relative-time')!.attributes.datetime.value, commitUrl];
 	}
 
-	const relativeTime = await fetchDom(
+	const commit = await fetchDom(
 		`${getRepoURL()}/commits?after=${commitSha}+${commitsCount - 2}`,
-		'.commit-meta relative-time'
+		'.commit'
 	);
-
-	return relativeTime!.attributes.datetime.value;
+	const timeStamp = select('relative-time', commit)!.attributes.datetime.value;
+	const {pathname} = select<HTMLAnchorElement>('a.message', commit)!;
+	return [timeStamp, pathname];
```

Comment:
Kinda:
```suggestion
	return [new Date(timeStamp).getTime(), pathname];
```

Reply:
We need the date for the title, so we cant just store the time